tests/unit-compatibility-test/fixtures/sonarqube-dce/secret-values.yaml

```diff
@@ -223,7 +223,7 @@ metadata:
     heritage: Helm
 data:
   SONAR_JDBC_USERNAME: sonarUser
-  SONAR_JDBC_URL: "jdbc:postgresql://%!s(<nil>):5432/sonarDB"
+  SONAR_JDBC_URL: "jdbc:postgresql://:5432/sonarDB"
```

Comment:
here this highlight a problem we had before, in this test case we have no jdbcoverwrite as well as postgresql.enabled=false.

Hence the default data in the configmap does not make sense.

i am creating a hardening ticket to complete the migration to having config map, secret and so on, created only when it make sense.

Here for example the configmap should exist only if jdbcoverwrite or postgresql.enabled, and should fail fast in dce